test/LicenseHeaderTest.ts

```diff
@@ -43,7 +38,21 @@ describe("LicenseHeaderCheck", function() {
                 }
             }
 
-            if (!contents.startsWith(license)) {
+            const match = licenseRegEx.exec(contents);
+            let validLicenseRange = false;
+            if (match !== null && match.length === 3) {
+                const start = parseInt(match[1], 10);
+                if (match[2] !== undefined) {
+                    // Year is a range
+                    const end = parseInt(match[2], 10);
+                    validLicenseRange =
+                        !isNaN(start) && !isNaN(end) && start >= 2017 && start < end && end <= 2020;
```

Comment:
can you somehow get the current year instead of end <= 2020?

Reply:
Maybe just use:
```
end <= new Date().getFullYear();
```

Reply:
done